common/app/views/fragments/head.scala.html

```diff
@@ -14,6 +14,22 @@
     <script src="@Static("javascripts/components/html5shiv/html5shiv.js")"></script>
 <![endif]-->
 
+@* provide some ipad users with the core view of fronts *@
+<script>
```

Comment:
Does this "random" choice persist? Will people get the same thing every time they show up?